site/src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -214,3 +216,23 @@ export const DropdownMenuShortcut = ({
 	);
 };
 DropdownMenuShortcut.displayName = "DropdownMenuShortcut";
+
+export interface DropdownMenuButtonProps
+	extends ComponentPropsWithoutRef<typeof Button> {}
+
+export const DropdownMenuButton = forwardRef<
```

Comment:
Hm... I hadn’t thought of that 🤔. Honestly—maybe it’s just personal—I don’t see any confusion since a trigger is a behavioral component and a button is a visual component. To me, it feels okay to have both. What would you suggest?

Reply:
@BrunoQuaresma It looks like you are adding DropdownMenuButton to the shadcn DropdownMenu component? Im curious why it is necessary to add this?

Reply:
The purpose of this component is to make dropdown button styling consistent. For example, instead of composing a `Button` + `Icon` + styling in every component or page where you want to add a dropdown, you could use a consistent component to reduce boilerplate. While I don’t mind boilerplate in general, in this case, since we already have a consistent design, it could help avoid inconsistencies.

**Before:**

```tsx
<Button ref={ref} variant="outline">
	Admin settings
	<DropdownMenuChevronDown />
</Button>
```

**After:**

```tsx
<DropdownMenuButton>Admin settings</DropdownMenuButton>
```

However, while writing this, I thought of another approach that might be better—creating a new `dropdown` variant in the `Button` component:

```tsx
<Button ref={ref} variant="dropdown">
	Admin settings
</Button>
```

Thoughts?

Reply:
If every DropdownMenu component will always use the DropdownMenuButton with this styling then it could make sense. However. that may not always be the case. I personally like having a dropdown variant for the button component more than altering the DropdownMenu component.

Reply:
I decided to remove the `DropdownMenuButton` component for now until we actually need it.